mate-session/gsm-manager.c

```diff
@@ -3376,14 +3340,6 @@ gsm_manager_can_shutdown (GsmManager *manager,
 #ifdef HAVE_SYSTEMD
         GsmSystemd *systemd;
 #endif
-        gboolean can_suspend = FALSE;
```

Comment:
How did I miss this... removing this code block breaks build with systemd enabled :open_mouth: 

Well, the logic was flawed anyway. We were asking upower if we can suspend/hibernate when we could ask systemd instead. I'll correct it.